### PR TITLE
6.7 bug fixes

### PR DESCRIPTION
**Pull request by stas-vilchik**
One more commit to come...

### CODE REVIEW

File: server/sonar-web/src/main/js/apps/portfolio/components/App.tsx [hunk 1]
```diff
@@ -152,13 +152,13 @@ export default class App extends React.PureComponent<Props, State> {
         </div>
 
         {subComponents != undefined &&
```
Comment (SonarTech) [inline server/sonar-web/src/main/js/apps/portfolio/components/App.tsx]: MAJOR != should be !== [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1440)

File: server/sonar-web/src/main/js/apps/portfolio/components/App.tsx [hunk 2]
```diff
@@ -152,13 +152,13 @@ export default class App extends React.PureComponent<Props, State> {
...
-        totalSubComponents != undefined && (
-          <WorstProjects
-            component={component.key}
-            subComponents={subComponents}
-            total={totalSubComponents}
-          />
-        )}
+          totalSubComponents != undefined && (
```
Comment (SonarTech) [inline server/sonar-web/src/main/js/apps/portfolio/components/App.tsx]: MAJOR != should be !== [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1440)

**Comment by SonarTech** [comment]
SonarQube analysis reported 5 issues
* ![MAJOR][MAJOR] 5 major

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [App.tsx#L101](https://github.com/SonarSource/sonarqube/blob/da2c362f560041747a6325658679f2e3c9e2fa00/server/sonar-web/src/main/js/apps/portfolio/components/App.tsx#L101): == should be === [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1440)
1. ![MAJOR][MAJOR] [App.tsx#L101](https://github.com/SonarSource/sonarqube/blob/da2c362f560041747a6325658679f2e3c9e2fa00/server/sonar-web/src/main/js/apps/portfolio/components/App.tsx#L101): == should be === [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1440)
1. ![MAJOR][MAJOR] [App.tsx#L104](https://github.com/SonarSource/sonarqube/blob/da2c362f560041747a6325658679f2e3c9e2fa00/server/sonar-web/src/main/js/apps/portfolio/components/App.tsx#L104): == should be === [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1440)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: server/sonar-web/src/main/js/apps/quality-profiles/home/CreateProfileForm.tsx [hunk 3]
```diff
@@ -176,6 +176,8 @@ export default class CreateProfileForm extends React.PureComponent<Props, State>
                   </p>
                 </div>
               ))}
+              {/* drop me when we stop supporting ie11 */}
+              <input type="hidden" name="hello-ie11" value="" />
```
Comment (gregaubert) [inline server/sonar-web/src/main/js/apps/quality-profiles/home/CreateProfileForm.tsx]: :hand: Hello IE11! :rofl:

File: server/sonar-web/src/main/js/api/permissions.ts [hunk 4]
```diff
@@ -178,12 +178,16 @@ export function removeProjectCreatorFromTemplate(
...
+  login: string;
+}
+
 export function getPermissionsUsersForComponent(
   projectKey: string,
   query?: string,
   permission?: string,
   organization?: string
-): Promise<any> {
+): Promise<PermissionUser[]> {
```
Comment (gregaubert) [inline server/sonar-web/src/main/js/api/permissions.ts]: Currently `PermissionUser` isn't used anywhere else. Why not just `Promise<{ login:string }[]>` ?

File: server/sonar-web/src/main/js/apps/projectsManagement/__tests__/ProjectRow-test.tsx [hunk 5]
```diff
@@ -44,17 +43,11 @@ it('checks project', () => {
   expect(onProjectCheck).toBeCalledWith(project, false);
 });
 
-it('applies permission template', () => {
```
Comment (gregaubert) [inline server/sonar-web/src/main/js/apps/projectsManagement/__tests__/ProjectRow-test.tsx]: Could be nice to create a test for `ProjectRowActions` and add that test back there.

**Pull request merged**